com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/Editor/ShaderGraph/Targets/BuiltInTarget.cs

```diff
@@ -331,8 +379,30 @@ void UpgradeAlphaClip()
 
         public override bool WorksWithSRP(RenderPipelineAsset scriptableRenderPipeline)
         {
-            return false;
+            return scriptableRenderPipeline == null;
         }
+
+        #region Metadata
+        string IHasMetadata.identifier
+        {
+            get
+            {
+                // defer to subtarget
+                if (m_ActiveSubTarget.value is IHasMetadata subTargetHasMetaData)
+                    return subTargetHasMetaData.identifier;
+                return null;
+            }
+        }
+
+        ScriptableObject IHasMetadata.GetMetadataObject()
+        {
+            // defer to subtarget
+            if (m_ActiveSubTarget.value is IHasMetadata subTargetHasMetaData)
+                return subTargetHasMetaData.GetMetadataObject();
+            return null;
+        }
+
+        #endregion
     }
 
     // XXXjesseb - Fix this!  Universal references.
```

Comment:
This comment can go away (doesn't have to as part of this PR, but it's no longer relevant).